README.md

```diff
@@ -32,7 +32,9 @@ Setting up the reference app is easy, just follow these steps:
     $ cp .env.example .env
     ```
 
-    **Enter values for `NEXT_PUBLIC_PROJECT_ID`, `NEXT_PUBLIC_PROJECT_DID` and `NEXT_PUBLIC_API_KEY_HASH` from your Affinidi project properties.**  
+    **Enter values for `NEXT_PUBLIC_PROJECT_ID`, `NEXT_PUBLIC_PROJECT_DID` and `NEXT_PUBLIC_API_KEY_HASH` from your Affinidi project properties.**
+
+    For all aps u can find two variables: `ISSUER_LOGIN` and `ISSUER_HASH_PASSWORD`. This is only an example of usage, and it should be replaced with appropriate auth mechanism (e.g. OAuth or any other match your requirements). 
```

Comment:
Provide a simple example on how to generate the hashed value and the salt

Reply:
Not sure we need to provide an example of hash generation due to the comment about hash should be replaced with the appropriate auth mechanism

Reply:
If I don't want to replace it with appropriate mechanism but still want to customize the password, I think a small example wouldn't hurt :D

Reply:
updated